dev-qt/qttools/qttools-6.9999.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qt6-build
+
+DESCRIPTION="Qt Tools Collection"
+
+if [[ ${QT6_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64"
+fi
+
+IUSE="
+	assistant designer distancefieldgenerator +linguist pixeltool
+	qattributionsscanner qdbus qdiag qdoc qplugininfo
+"
+
+# behaves very badly when qttools is not already installed, also
+# other more minor issues (clang tests flaky depending on version,
+# and 3rdparty/ tries to FetchContent gtest)
+RESTRICT="test"
+
+RDEPEND="
+	=dev-qt/qtbase-${PV}*:6[network]
+	assistant? ( =dev-qt/qtbase-${PV}*:6[sql,widgets] )
+	designer? ( =dev-qt/qtbase-${PV}*:6[widgets] )
+	distancefieldgenerator? (
+		=dev-qt/qtbase-${PV}*:6[widgets]
+		=dev-qt/qtdeclarative-${PV}*:6
+	)
+	pixeltool? ( =dev-qt/qtbase-${PV}*:6[widgets] )
+	qdbus? ( =dev-qt/qtbase-${PV}*:6[widgets] )
+	qdiag? ( =dev-qt/qtbase-${PV}*:6[opengl,widgets] )
+	qdoc? ( sys-devel/clang:= )
```

Comment:
Do we consider libclang usage without llvm.eclass to be smelly?

Reply:
That's already in my TODO (I didn't get to qttools yet), not marked as draft for nothing

Reply:
Also probably need to have a USE=clang/llvm or so, given clang can be used for other things it seems too and I don't think merging it with USE=qdoc will keep working out.